### ISSUE TITLE
Add type: mount|env options to --secrets.

### ISSUE THREAD

**Issue by schmidthausen** [containers/ansible-podman-collections#362]
**Is this a BUG REPORT or FEATURE REQUEST? (leave only one on its own line)**

/kind feature

**Description**

**Steps to reproduce the issue:**

n/a

**Describe the results you received:**
n/a

**Describe the results you expected:**
n/a

**Additional information you deem important (e.g. issue happens only occasionally):**
n/a

**Version of the `containers.podman` collection:**
**Either git commit if installed from git: `git show --summary`**
**Or version from `ansible-galaxy` if installed from galaxy: `ansible-galaxy collection list | grep containers.podman`**

```
containers.podman       1.9.0
```

**Output of `ansible --version`:**

```
ansible [core 2.12.1]
  config file = None
  configured module search path = ['/home/john/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /home/john/.local/lib/python3.10/site-packages/ansible
  ansible collection location = /home/john/.ansible/collections:/usr/share/ansible/collections
  executable location = /home/john/.local/bin/ansible
  python version = 3.10.1 (main, Dec  9 2021, 00:00:00) [GCC 11.2.1 20211203 (Red Hat 11.2.1-7)]
  jinja version = 3.0.3
  libyaml = True
```

**Output of `podman version`:**

```
podman version 3.4.4
```

**Output of `podman info --debug`:**

``` yaml
not required - feature request
```

**Package info (e.g. output of `rpm -q podman` or `apt list podman`):**

```
podman-3.4.4-1.fc35.x86_64
```

**Playbok you run with ansible (e.g. content of `playbook.yaml`):**

``` yaml
not required - feature request
```

**Command line and output of ansible run with high verbosity**

**Please NOTE: if you submit a bug about idempotency, run the playbook with `--diff` option, like:**

`ansible-playbook -i inventory --diff -vv playbook.yml`

```
not required - feature request
```

**Additional environment details (AWS, VirtualBox, physical, etc.):**

**Comment by schmidthausen** [comment]
Secret Options
    type=mount|env : How the secret will be exposed to the container. Default mount.
    target=target : Target of secret. Defaults to secret name.
    uid=0 : UID of secret. Defaults to 0. Mount secret type only.
    gid=0 : GID of secret. Defaults to 0. Mount secret type only.
    mode=0 : Mode of secret. Defaults to 0444. Mount secret type only.

**Comment by sshnaidm** [comment]
@amenzhinsky hey, would you like to look into this?

**Comment by amenzhinsky** [comment]
@sshnaidm can do later this weak

**Comment by amenzhinsky** [comment]
This is already possible:
```yaml
secrets:
  - mysecret,type=env,target=SECRET
```
As an enhancement `secrets` could accept list of strings or dicts, like:
```yaml
secrets:
  - foo
  - name: mysecret
    type: env
```
Not sure if it's really worth it.

**Comment by sshnaidm** [comment]
@schmidthausen does it seem ok for you?

**Comment by schmidthausen** [comment]
On Thursday, January 6, 2022 5:05:37 A.M. PST Sergey wrote:
> @schmidthausen does it seem ok for you?
Yes, this is great! I was reading the docs incorrectly. I got this to work, 
thank you for the response.

**Comment by sshnaidm** [comment]
Fixed by #363
